src/component/project/Project/EditProject/EditProject.tsx

```diff
@@ -68,6 +73,12 @@ const EditProject = () => {
         navigate(-1);
     };
 
+    const accessDeniedAlert = !hasAccess(UPDATE_PROJECT, projectId) && (
+        <Alert severity="error" sx={{ mb: 4 }}>
+            You do not have access to edit this project.
```

Comment:
Maybe this should be permission instead?

```suggestion
            You do not have permission to edit this project.
```

Reply:
Or maybe even:

```suggestion
            You do not have the required permissions to edit this project.
```